addon/components/rdf-input-fields/concept-scheme-selector.js

```diff
@@ -33,17 +37,40 @@ export default class RdfInputFieldsConceptSchemeSelectorComponent extends InputF
   loadOptions() {
     const metaGraph = this.args.graphs.metaGraph;
     const fieldOptions = this.args.field.options;
+    let conceptScheme = null;
 
     if (!hasValidFieldOptions(this.args.field, ['conceptScheme'])) {
-      return;
-    }
+      const scheme = this.args.formStore.any(
+        this.args.field.uri,
+        FORM_OPTION('conceptScheme'),
+        undefined,
+        this.args.graphs.formGraph
+      );
+      if (!scheme) {
+        return;
+      }
 
-    const conceptScheme = new namedNode(fieldOptions.conceptScheme);
+      conceptScheme = scheme;
+    } else {
+      conceptScheme = new namedNode(fieldOptions.conceptScheme);
+    }
 
     /**
      * NOTE: Most forms are now implemented to have a default "true" behavior
      */
-    if (fieldOptions.searchEnabled !== undefined) {
+    if (!hasValidFieldOptions(this.args.field, ['searchEnabled'])) {
+      const isEnabled = this.args.formStore.any(
+        this.args.field.uri,
+        FORM_OPTION('searchEnabled'),
+        undefined,
+        this.args.graphs.formGraph
+      );
+      if (!isEnabled) {
+        return;
+      }
+
+      this.searchEnabled = isEnabled == '1' ? true : false;
```

Comment:
Could convert it depending on datatype

Reply:
You could also use the `Literal.toJS` util I think: http://linkeddata.github.io/rdflib.js/doc/classes/Literal.html#toJS

Reply:
Yes nice one works great